packages/app-claims/src/index.tsx

```diff
@@ -124,7 +124,13 @@ class App extends TxModal<Props, State> {
                   chain: system_chain.toString()
                 }
               })}</h3>
-              {this.renderInputAccount()}
+              <InputAddress
+                defaultValue={this.state.accountId}
+                help={t('The account you want to claim to.')}
+                label={t('claim to account')}
+                onChange={this.onChangeAccount}
+                type='allPlus'
```

Comment:
Rather just `all`, `allPlus` has contracts as well.

```suggestion
                type='all'
```